<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DDPosBurgCircuitResults</class>
 <widget class="QDialog" name="DDPosBurgCircuitResults">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>621</width>
    <height>651</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Results of the calculation of the dislocation position</string>
  </property>
  <property name="windowFilePath">
   <string/>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>610</y>
     <width>171</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QCheckBox" name="showDPoly">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>580</y>
     <width>471</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Show the boundary of the dislocation region in the main window</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="tristate">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>581</width>
     <height>251</height>
    </rect>
   </property>
   <property name="title">
    <string>Computed in the deformed configuration</string>
   </property>
   <widget class="QTableWidget" name="coorNodes_rel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>214</width>
      <height>171</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="toolTip">
     <string>positions of atoms in units of the lattice parameter in the unrelaxed and relaxed configurations</string>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="gridStyle">
     <enum>Qt::SolidLine</enum>
    </property>
    <property name="rowCount">
     <number>0</number>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>true</bool>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>98</number>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderDefaultSectionSize">
     <number>20</number>
    </attribute>
    <column>
     <property name="text">
      <string>x</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>y</string>
     </property>
    </column>
   </widget>
   <widget class="QGroupBox" name="groupBox1_1">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>60</y>
      <width>311</width>
      <height>171</height>
     </rect>
    </property>
    <widget class="QLabel" name="ydrel_tol">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>50</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>0.0000</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>50</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>yD =</string>
     </property>
    </widget>
    <widget class="QLabel" name="xdrel_tol">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>30</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>0.0000</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>191</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Position of the dislocation:</string>
     </property>
    </widget>
    <widget class="QLabel" name="pm2">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>50</y>
       <width>21</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>+/-</string>
     </property>
    </widget>
    <widget class="QLabel" name="pm1">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>30</y>
       <width>21</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>+/-</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>30</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>xD =</string>
     </property>
    </widget>
    <widget class="QLabel" name="xdrel">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>30</y>
       <width>81</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>+0.0000</string>
     </property>
    </widget>
    <widget class="QLabel" name="ydrel">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>50</y>
       <width>81</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>+0.0000</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>261</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Burgers vector inside of the polygon:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>100</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Bx =</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>120</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>By =</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>140</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Bz =</string>
     </property>
    </widget>
    <widget class="QLabel" name="Bx">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>100</y>
       <width>81</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>+0.0000</string>
     </property>
    </widget>
    <widget class="QLabel" name="By">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>120</y>
       <width>81</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>+0.0000</string>
     </property>
    </widget>
    <widget class="QLabel" name="Bz">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>140</y>
       <width>81</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>+0.0000</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>251</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Boundary of the dislocation region:</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>280</y>
     <width>581</width>
     <height>281</height>
    </rect>
   </property>
   <property name="title">
    <string>Mapping to the initial (undeformed) configuration</string>
   </property>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>251</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Boundary of the dislocation region:</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox2_2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>70</y>
      <width>311</width>
      <height>81</height>
     </rect>
    </property>
    <widget class="QLabel" name="ydinit_tol">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>50</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>0.0000</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>50</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>YD =</string>
     </property>
    </widget>
    <widget class="QLabel" name="xdinit_tol">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>30</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>0.0000</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>191</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Position of the dislocation:</string>
     </property>
    </widget>
    <widget class="QLabel" name="pm2_2">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>50</y>
       <width>21</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>+/-</string>
     </property>
    </widget>
    <widget class="QLabel" name="pm1_2">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>30</y>
       <width>21</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>+/-</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>30</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>XD =</string>
     </property>
    </widget>
    <widget class="QLabel" name="xdinit">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>30</y>
       <width>81</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>+0.0000</string>
     </property>
    </widget>
    <widget class="QLabel" name="ydinit">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>50</y>
       <width>81</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>+0.0000</string>
     </property>
    </widget>
   </widget>
   <widget class="QTableWidget" name="coorNodes_init">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>214</width>
      <height>171</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="toolTip">
     <string>positions of atoms in units of the lattice parameter in the unrelaxed and relaxed configurations</string>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="gridStyle">
     <enum>Qt::SolidLine</enum>
    </property>
    <property name="rowCount">
     <number>0</number>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>true</bool>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>98</number>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderDefaultSectionSize">
     <number>20</number>
    </attribute>
    <column>
     <property name="text">
      <string>X</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Y</string>
     </property>
    </column>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DDPosBurgCircuitResults</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DDPosBurgCircuitResults</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
